src/hugr/rewrite/consts.rs

```diff
@@ -0,0 +1,231 @@
+//! Rewrite operations involving Const and LoadConst operations
+
+use std::iter;
+
+use crate::{
+    hugr::{HugrError, HugrMut},
+    HugrView, Node,
+};
+#[rustversion::since(1.75)] // uses impl in return position
+use itertools::Itertools;
+use thiserror::Error;
+
+use super::Rewrite;
+
+/// Remove a [`crate::ops::LoadConstant`] node with no outputs.
+#[derive(Debug, Clone)]
+pub struct RemoveConstIgnore(pub Node);
```

Comment:
We should really rename this to `RemoveLoadConstant` rather than this "ignore" malarky?

Reply:
made issue #767